spikeinterface/sorters/runsorter.py

```diff
@@ -144,17 +149,23 @@ class ContainerClient:
       * docker with "docker" python package
       * singularity with  "spython" python package
     """
-    def __init__(self, mode, container_image, volumes, extra_kwargs):
+    def __init__(self, mode, container_image, volumes, gpu_to_use=None):
```

Comment:
I would also leave the `extra_kwargs` since it allows for more flexibility (e.g. some custom bindings etc).

Reply:
Sure!

Reply:
Should be fixed with last commit!